bokun.yaml

```diff
@@ -5952,14 +6271,62 @@ definitions:
         description: 'Only used if hasOpeningHours == true. The default opening hours by weekday. These may be overridden by seasonal opening hours.'
       seasonalOpeningHours:
         type: array
-        description: 'Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
+        description: 'Deprecated. Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
         items:
           $ref: '#/definitions/SeasonalOpeningHours'
+      displaySettings:
+        type: object
+        properties:
+          showPickupPlaces:
+            type: boolean
+            description: 'Specifies if pick up places are shown.'
+          showRouteMap:
+            type: boolean
+          selectRateBasedOnStartTime:
+            type: boolean
+          customFields:
+            type: array
+            items:
+              type: object
+            description: 'Deprecated.'
```

Comment:
why is this deprecated?

Reply:
Do not remember for sure, why documented "customFields" (property of "displaySettings") as deprecated, but I could not find how to get elements into this array. 
By the way, custom field was created for tested activity and was shown in another  "customFields" (which is property of ActivityDto).

Reply:
Works from old settings, fixed.